pkg/kubectl/proxy_server.go

```diff
@@ -35,7 +35,7 @@ import (
 const (
 	DefaultHostAcceptRE   = "^localhost$,^127\\.0\\.0\\.1$,^\\[::1\\]$"
 	DefaultPathAcceptRE   = "^/.*"
-	DefaultPathRejectRE   = "^/api/.*/exec,^/api/.*/run,^/api/.*/attach"
```

Comment:
What do you think of just tightening to:

>
^/api/.*/pods/.*/exec
^/api/.*/pods/.*/attach

I think this will address the issues without opening it up more than is needed.